deployment/keystone/changeset/operations/contracts/deploy_contracts_seq.go

```diff
@@ -59,13 +66,29 @@ var DeployKeystoneContractsSequence = operations.NewSequence[DeployKeystoneContr
 		as := datastore.NewMemoryDataStore()
 
 		// Capabilities Registry contract
-		capabilitiesRegistryDeployReport, err := operations.ExecuteOperation(b, DeployCapabilityRegistryOp, DeployCapabilityRegistryOpDeps(deps), DeployCapabilityRegistryInput{ChainSelector: input.RegistryChainSelector})
-		if err != nil {
-			return DeployKeystoneContractsSequenceOutput{}, err
-		}
-		err = updateAddresses(as.Addresses(), capabilitiesRegistryDeployReport.Output.Addresses, ab, capabilitiesRegistryDeployReport.Output.AddressBook)
-		if err != nil {
-			return DeployKeystoneContractsSequenceOutput{}, err
+		if input.WithV2Contracts {
+			v2Report, err := operations.ExecuteOperation(b, cap_reg_v2.DeployCapabilitiesRegistry, cap_reg_v2.DeployCapabilitiesRegistryDeps(deps), cap_reg_v2.DeployCapabilitiesRegistryInput{
+				ChainSelector: input.RegistryChainSelector,
+			})
+			if err != nil {
+				return DeployKeystoneContractsSequenceOutput{}, err
+			}
+			
+			out, err := ToV1Output(v2Report.Output)
+			if err != nil {
+				return DeployKeystoneContractsSequenceOutput{}, err
+			}
```

Comment:
would it make sense to have a generic output, which would either be set to v1 or v2 output report?